### DISCUSSION TITLE
8BitDo Retro Mechanical Keyboard not working on GameCube!

### DISCUSSION THREAD

**Discussion by ABOhiccups** [darthcloud/BlueRetro#786] [Q&A]
I just got an 8BitDo Retro Mechanical Keyboard. I tested Wired and 2.4Ghz Wireless on Windows. It works great! But there is one problem. I could not get the keyboard to work on Nintendo GameCube through BlueRetro Internal Board.

I go to BlueRetro Web UI and set the preset on Port Number 2 to Gamepad/Keyboard. 8BitDo Retro Mechanical Keyboard is still not responding in Phantasy Star Online Episode I & II for GameCube.

Anyone else has same issue with 8BitDo Retro Mechanical Keyboard?

**Comment by darthcloud** [comment]
I don't have one but KB support was broken in v1.8.4,

you can try this debug build and see if it work:
https://github.com/darthcloud/BlueRetro/discussions/775#discussioncomment-7143711

**Comment by ABOhiccups** [comment]
I updated to Debug Build. I set Preset on Output 2 to Gamepad/Keyboard. Nothing happens!

**Comment by ABOhiccups** [comment]
8BitDo Retro Mechanical Keyboard is not compatible with v1.9 HW2.
